embassy-stm32/src/dac/mod.rs

```diff
@@ -51,7 +51,10 @@ impl Ch1Trigger {
     fn tsel(&self) -> dac::vals::Tsel1 {
         match self {
             Ch1Trigger::Tim6 => dac::vals::Tsel1::TIM6_TRGO,
+            #[cfg(not(dac_v3))]
             Ch1Trigger::Tim3 => dac::vals::Tsel1::TIM3_TRGO,
+            #[cfg(dac_v3)]
+            Ch1Trigger::Tim3 => dac::vals::Tsel1::TIM1_TRGO,
```

Comment:
this seems wrong

Reply:
@JuliDi will fix it in the next pr

Reply:
This should do:
```rust
#[derive(Debug, Copy, Clone, Eq, PartialEq)]
#[cfg_attr(feature = "defmt", derive(defmt::Format))]
/// Trigger sources for CH1
pub enum Ch1Trigger {
    #[cfg(dac_v3)]
    Tim1,
    Tim2,
    #[cfg(dac_v2)]
    Tim3,
    #[cfg(dac_v3)]
    Tim4,
    #[cfg(dac_v3)]
    Tim5,
    Tim6,
    Tim7,
    #[cfg(dac_v3)]
    Tim8,
    Tim15,
    #[cfg(dac_v3)]
    Hrtim1Dactrg1,
    #[cfg(dac_v3)]
    Hrtim1Dactrg2,
    #[cfg(dac_v3)]
    Lptim1,
    #[cfg(dac_v3)]
    Lptim2,
    #[cfg(dac_v3)]
    Lptim3,
    Exti9,
    Software,
}

impl Ch1Trigger {
    fn tsel(&self) -> dac::vals::Tsel1 {
        match self {
            #[cfg(dac_v3)]
            Ch1Trigger::Tim1 => dac::vals::Tsel1::TIM1_TRGO,
            Ch1Trigger::Tim2 => dac::vals::Tsel1::TIM2_TRGO,
            #[cfg(dac_v2)]
            Ch1Trigger::Tim3 => dac::vals::Tsel1::TIM3_TRGO,
            #[cfg(dac_v3)]
            Ch1Trigger::Tim4 => dac::vals::Tsel1::TIM4_TRGO,
            #[cfg(dac_v3)]
            Ch1Trigger::Tim5 => dac::vals::Tsel1::TIM5_TRGO,
            Ch1Trigger::Tim6 => dac::vals::Tsel1::TIM6_TRGO,
            Ch1Trigger::Tim7 => dac::vals::Tsel1::TIM7_TRGO,
            #[cfg(dac_v3)]
            Ch1Trigger::Tim8 => dac::vals::Tsel1::TIM8_TRGO,
            Ch1Trigger::Tim15 => dac::vals::Tsel1::TIM15_TRGO,
            #[cfg(dac_v3)]
            Ch1Trigger::Hrtim1Dactrg1 => dac::vals::Tsel1::HRTIM1_DACTRG1,
            #[cfg(dac_v3)]
            Ch1Trigger::Hrtim1Dactrg2 => dac::vals::Tsel1::HRTIM1_DACTRG2,
            #[cfg(dac_v3)]
            Ch1Trigger::Lptim1 => dac::vals::Tsel1::LPTIM1_OUT,
            #[cfg(dac_v3)]
            Ch1Trigger::Lptim2 => dac::vals::Tsel1::LPTIM2_OUT,
            #[cfg(dac_v3)]
            Ch1Trigger::Lptim3 => dac::vals::Tsel1::LPTIM3_OUT,
            Ch1Trigger::Exti9 => dac::vals::Tsel1::EXTI9,
            Ch1Trigger::Software => dac::vals::Tsel1::SOFTWARE,
        }
    }
}
```

Reply:
But I'll add it to the next PR. And @Dirbaio is right, it should not be v3 for the `Tim3` but v2